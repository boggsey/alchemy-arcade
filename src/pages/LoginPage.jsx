import React from 'react';
import Login from '../components/Account/Login';

const LoginPage = () => (
  <div>
    <Login />
  </div>
);

export default LoginPage;
