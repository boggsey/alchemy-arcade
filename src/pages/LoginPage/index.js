import React from 'react';
import Login from '../../components/Account/Login/Login.component';

const LoginPage = () => (
  <Login />
);

export default LoginPage;
