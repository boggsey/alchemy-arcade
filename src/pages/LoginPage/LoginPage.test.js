import React from 'react';
import { shallow } from 'enzyme';
import LoginPage from '.';

describe('LoginPage', () => {
  it('Hasn\'t changed since the last snapshot', () => {
    const wrapper = shallow(<LoginPage isAuthenticated receiveLogout={() => {}} />);

    expect(wrapper).toMatchSnapshot();
  });
});
