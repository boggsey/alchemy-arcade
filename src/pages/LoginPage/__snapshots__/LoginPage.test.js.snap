// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginPage Hasn't changed since the last snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <LoginPage
    isAuthenticated={true}
    receiveLogout={[Function]}
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {},
    "ref": null,
    "rendered": null,
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {},
      "ref": null,
      "rendered": null,
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
